src/VisualStudio/Core/Test/Venus/DocumentService_IntegrationTests.vb

```diff
@@ -55,12 +55,9 @@ class {|Definition:C1|}
     </Project>
 </Workspace>
 
-            Dim exportProvider = ExportProviderCache _
-                .GetOrCreateExportProviderFactory(TestExportProvider.EntireAssemblyCatalogWithCSharpAndVisualBasic() _
-                .WithParts(GetType(MockServiceProvider))) _
-                .CreateExportProvider()
+            Dim composition = EditorTestCompositions.EditorFeatures.AddParts(GetType(MockServiceProvider))
```

Comment:
💭 Seems like this should use a Visual Studio composition, since `IServiceProvider` is only applicable in that context.

Reply:
It should, but it's not working. Filed https://github.com/dotnet/roslyn/issues/46279 to follow up.